documentatie/ontwikkelproces/zero-bug-policy.md

```diff
@@ -0,0 +1,18 @@
+# Uitwerking zero bug policy
+
+Dit document beschrijft de praktische uitwerking van de [zero bug policy](/documentatie/ontwikkelproces/testen-en-kwaliteit.md#zero-bug-policy) vermeld in "Testen en kwaliteit".
+
+Een bug is iets wat de waarde van onze applicatie in gevaar brengt. Dit is dus een brede definitie, over alle relevante [kwaliteitsattributen](/documentatie/ontwikkelproces/testen-en-kwaliteit.md#kwaliteitsattributen) heen. De ernst van een bug kan variëren van "nu gelijk fixen" tot "nooit fixen, nergens bijhouden".
+
+Het doel van de zero bug policy is vermijden dat we een backlog verzamelen met dingen die we nog moeten verbeteren, aan werk dat eigenlijk af is.
+
+Het gevaar van de zero bug policy is dat pull requests (PRs) heel lang open blijven, omdat we pas mergen als alle puntjes op alle i's staan. Dan optimaliseer je voor kwaliteit in de PR-branch ten koste van meer risico bij code-integratie (mergen naar main). Da's niet verstandig.
```

Comment:
```suggestion
Het gevaar van de zero bug policy is dat pull requests (PRs) heel lang open blijven, omdat we pas mergen als alle puntjes op alle i's staan. Dan optimaliseer je voor kwaliteit in de PR-branch ten koste van meer risico bij code-integratie (mergen naar main). Dat niet verstandig.
```
Los van de spelling: ik snap deze alinea niet zo goed. Ik begrijp wel wat er staat, maar de conclusie 'dat is niet verstandig' komt een beetje uit de lucht vallen. En er mist een 'Hoe dan wel?' 

In 'Dan optimaliseer je...' is het niet helemaal duidelijk waar de 'dan' naar terugverwijst.

Reply:
@jorisleker Is dit beter?

"Het doel van de zero bug policy is vermijden dat we een backlog verzamelen met dingen die we nog moeten verbeteren aan werk dat eigenlijk af is.

Als we die policy echter heel streng toepassen op elk pull request (PR), dan blijven onze pull requests open staan tot alle puntjes op alle i's staan. Je optimaliseet dan voor kwaliteit in de PR-branch te koste van meer risico bij code-integratie (mergen naar main). De truc is dus om de balans te bewaren tussen PRs binnen 1-3 dagen mergen en geen bochten afsnijden qua kwaliteit. 

Hiervoor passen we zero bug policy trapsgewijs toe over epics, issues en PRs heen:"

Reply:
Ja beter. Paar spellingsuggesties nog, scheelt straks weer een request for change :-)

"We hanteren een _zero bug policy_ om te vermijden dat we een verzameling maken van dingen die we nog moeten verbeteren aan werk dat eigenlijk al af is.

Als we dat heel streng toepassen op elk pull request (PR), dan blijven onze PRs openstaan tot alle puntjes op alle i's staan. We optimaliseren dan voor kwaliteit in de PR-branch te koste van meer risico bij code-integratie (mergen naar main). De truc is dus om de balans te bewaren tussen PRs binnen 1-3 dagen mergen en geen bochten afsnijden qua kwaliteit.

Daarom passen we zero bug policy trapsgewijs toe over epics, issues en PRs heen:"

Reply:
Overgenomen met een kleine wijziging in de eerste zin. Goed punt om "backlog" niet te gebruiken.